packages/spindle-ui/src/Button/Button.css

```
@@ -4,8 +4,7 @@
 */
 :root {
   --Button-tapHighlightColor: var(--gray-5-alpha);
-  --Button-onFocus-boxShadow: 0 0 0 1px var(--color-surface-primary),
-    0 0 0 3px var(--color-focus-clarity);
+  --Button-onFocus-outlineColor: var(--color-focus-clarity);
```

少し細かいですが

```
  --Button--outlined-borderColor: var(--color-border-accent-primary);
  --Button--outlined-color: var(--color-text-accent-primary);
  --Button--outlined-onActive-backgroundColor: var(--primary-green-5);
  --Button--outlined-onHover-backgroundColor: var(--primary-green-5);
```

と定義されているので

```suggestion
  --Button-onFocus-outlinedColor: var(--color-focus-clarity);
```

の方が統一性あったりしますかね？

---

こり、非常に紛らわしいのですが、その位置はプロパティ名になるんです！なので、`outline-color`に対する文字列`outlineColor `を入れてます。ちなみに`outlined`はSpindle ButtonのVariant名称になります...!

---

そしたらこのままで良さそうですね！